AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+||sp-consent.stuttgarter-nachrichten.de^
```

Comment:
it breaks video(checked with German VPN)
https://www.stuttgarter-nachrichten.de/inhalt.video-schwaebische-maultaschenproduktion-rezept-zum-nachkochen-so-gelingen-original-schwaebische-maultaschen.edfc4068-2536-4e24-9e60-9c8cb97a3267.html

Reply:
Thank you. I will add a better rule. Just wait.

Reply:
@Alex-302 Fixed. Please check.